deepchecks/nlp/text_data.py

```diff
@@ -497,6 +497,17 @@ def is_sampled(self, n_samples: t.Optional[int]):
         if n_samples is None:
             return False
         return self.n_samples > n_samples
+    @classmethod
+    def from_hugging_face(cls, task_type:str, obj:t.Any, label: t.Optional[TTextLabel]=None ,metadata: t.Optional[pd.DataFrame]=None,
+            properties: t.Optional[pd.DataFrame] = None):
+
+        if task_type == 'token_classification':
+            return cls(tokenized_text=obj['tokens'], label=obj[label] if label is not None else obj['ner_tags'], task_type=task_type,metadata=metadata, properties=properties )
```

Comment:
Some comments:
1. The explicit "default" name should be defined as a global var for this file
2. There should be an if checking if the assumed fields are actually in the data, if not, an appropriate value error should be raised, telling the user to explicitly provide the fields. 
3. In case that the field exists, but are not of the required format (e.g. NER rags are spans and not IOB annotations), we should add a text to the resulting error message saying what fields we used by default, on top of the error message the deepchecks will produce for the illegal format.